partridge/geo.py

```diff
@@ -0,0 +1,38 @@
+from typing import Dict, List
+
+import pandas as pd
+
+try:
+    import geopandas as gpd
+    from shapely.geometry import LineString, Point
+except ImportError as impexc:
+    print(impexc)
+    print("You must install GeoPandas to use this module.")
+    exit(1)
+
+
+def build_shapes(df: pd.DataFrame) -> gpd.GeoDataFrame:
```

Comment:
It might be helpful to add error messages in case a user tries to use `build_shapes` or `build_stops` on something other than stops.txt, shapes.txt. However, I do imagine that the KeyError that they'd likely see would be sufficient in most cases.